src/pages/student-table/constants.js

```diff
@@ -0,0 +1,73 @@
+import Box from '@mui/material/Box'
+
+export const tabLabels = (t) => [
+  { label: t('studentTable.all'), value: null },
+  { label: t('studentTable.active'), value: true },
+  { label: t('studentTable.unactive'), value: false }
+]
+
+export const headCells = (t) => [
+  {
+    label: t('studentTable.name'),
+    id: 'firstName'
+  },
+  {
+    label: t('studentTable.email'),
+    id: 'email'
+  },
+  {
+    label: t('studentTable.role'),
+    id: 'role'
+  },
+  {
+    label: t('studentTable.lastLogin'),
+    id: 'lastLogin'
+  },
+  {
+    label: t('studentTable.firstLogin'),
+    id: 'isFirstLogin',
+    filterCheckboxesArr: [
+      {
+        label: t('studentTable.firstLogin'),
+        value: true
+      },
+      {
+        label: t('studentTable.notFirstLogin'),
+        value: false
+      }
+    ]
+  }
+]
+
+export const rowPropsArr = [
+  {
```

Comment:
dataField: 'firstName' - In this situation it shows the data from response.firstName

calculatedCellValue: (row) => row.firstName + row.lastName - if u need to return only calculated string value from couple of the fields.

cellTemplate: (row) => <CellComponent rowData={row} />

Reply:
done